### PR TITLE
Add support for 868 channel mapping

### PR DESCRIPTION
**Pull request by andymck**
Proto PR for https://github.com/helium/miner/pull/405

### CODE REVIEW

File: src/blockchain_state_channel_v1.proto [hunk 1]
```diff
@@ -39,6 +39,18 @@ message blockchain_state_channel_packet_v1 {
     packet packet = 1;
     bytes hotspot = 2;
     bytes signature = 3;
+    enum Region {
```
Comment (vihu) [inline src/blockchain_state_channel_v1.proto:42]: I don't fully understand why `Region` enum is defined within the message stanza?

Comment (Vagabond) [reply src/blockchain_state_channel_v1.proto:42]: Why not?

Comment (vihu) [reply src/blockchain_state_channel_v1.proto:42]: Just curious is all, if it's fine here then let's get this in.

Comment (Vagabond) [reply src/blockchain_state_channel_v1.proto:42]: merge it

**Pull request merged**